react/week2/todo/src/Todo.js

```diff
@@ -0,0 +1,81 @@
+
+import React, { useState } from 'react';
+import Item from './Item';
+import TodoForm from "./TodoForm";
+
+const Todo = () => {
+
+
+	const taskItem = [
```

Comment:
It's a small thing, but I would move the array out, as it's a constant, and you want to make sure to not re-create it any time the component is rendered.